wp-parsely.php

```diff
@@ -134,8 +134,8 @@ function parsely_admin_init_register(): void {
  * Network Admin Sites List table.
  */
 function parsely_wp_admin_early_register(): void {
-	$settings_page = new Settings_Page( $GLOBALS['parsely'] );
-	$settings_page->run();
+	$GLOBALS['parsely_settings_page'] = new Settings_Page( $GLOBALS['parsely'] );
```

Comment:
Using global variable because we are using this same instance for showing tabs on `views/parsely-settings.php`

Reply:
What does the transition to the global variable offer us? Was it required to make things work?

Reply:
I have to use this same instance in `views/parsely-settings.php` so therefore put it in globals.